### DISCUSSION TITLE
Lit review

### DISCUSSION THREAD

**Discussion by kbuzard** [ecn310/course-project-nepobabies#3] [General]
@ecn310/nepobabies I just checked out your Wiki and wanted to let you know something, but there's no way to comment on the Wiki and I didn't see where you're going to keep your lit review assignment so I just created this discussion.

The main thing I want to say is this: for tomorrow, it's okay if you only have five total papers as long as two of them are ones you found (so you can leave out two that Professor Zhu sent if you want).

Also, because of this issue with the wiki, it's going to work better to have your work plan/product (that you currently have on the Wiki) in an .md file, discussion or issue. Then you can just put a link to each output on the Wiki along with a short update. The basic issue is that there aren't tools on the wiki to interact and give feedback, so it's only really good as a straightforward record of what's been done.
- We can also brainstorm other documentation structures in class on Wednesday if you think there's something else that'll work better.

**Comment by rhrabino** [comment]
#### September 27, 2023
I have attached a link to the Google doc that contains our summary of the literature provided by Professor Zhu and some additional literature examined by me and Ryan. 
https://docs.google.com/document/d/1ohV_ooclwrWstYxeaZou1-57XIJWqJHV30ulo_EfT0U/edit?usp=sharing

**Comment by kbuzard** [comment]
@ecn310/nepobabies For now:
- I'm waiting on "edit" access to the Google doc.
- I'm also writing to remind you that you need to add to your high-level documentation about this task being accomplished, with a link to this discussion
   - The instructions from the assignment slide: "Add to your documentation wiki: that you worked on lit review today, where the material is, what you accomplished"
   - Note, this is just 1-2 sentences.

**Comment by kbuzard** [comment]
@ecn310/nepobabies I just left feedback on the google doc. I don't know your usernames to tag you there, so I'm tagging you here to let you know.

You have a lot of great material to work with. It just needs a little bit of refining for it to be useful this coming week when you work on formulating your research hypothesis.

**Comment by rhrabino** [comment]
### Lit Review 
#### October 31, 2023

This comment includes our Lit Review Plan from the end of September. We are moving it here as we repurpose our Wiki for master documentation.

### Lit Review Plan

**Read Greenlaw Chapter 6 before reading papers!**

#### Ryan
- Clearly state the research question.
- Read Oreopoulos (2012) and Kramarz & Skans (2014).
- Find one additional scholarly article.
- Write brief summaries for each article including information relevant to the research project.
- Include APA citations for each article.

#### Rachel
- Read Kahn (2010),Schwandt & von Wachter (2019), and Hellerstein & Morrill (2011).
- Find one additional scholarly article.
- Write brief summaries for each article including information relevant to the research project.
- Include APA citations for each article.

**Comment by ryanpseely** [comment]
### November 15, 2023
- A few days ago I found [this article](https://academic.oup.com/ej/article/133/652/1466/6998552?login=true) titled "Connecting the Young: High School Graduates’ Matching to First Jobs in Booms and Great Recessions." It analyzes labor market data across good times and bad times to find differences in using contacts for job matching.
- I was hoping that it would provide some support for the part of our theory that says during recessions, labor market entrants will do as much as they can to optimize their chances of getting a job. However, the theory essentially said that they can account for one-third of the variation in using social connections to find employees from the firm side, but the other two-thirds cannot be accounted for.
- @kbuzard said that we can still use this in our report, and say something like "We found that certain demographics, e.g. wealthy labor market entrants, are more likely to use their connections, even though economists did not find the same connection in Swedish labor market data," assuming that we actually find some sort of correlation with a moderating variable (demographic). If we do not find this correlation with any of the demographics, then we can report that, too, and say that we found similar results as this paper did. 
- For more specific analysis and quotes from the article, see the [google doc](https://docs.google.com/document/d/1ohV_ooclwrWstYxeaZou1-57XIJWqJHV30ulo_EfT0U/edit) where I took notes.
- [ ] As suggested by Professor Buzard, we should do a forward cite search and backward cite search using this paper.

**Comment by ryanpseely** [comment]
### November 17, 2023
- An idea I had for solving the problem we came across with the article mentioned above, that it did not find any results for labor market entrants using social connections to optimize their chances of getting jobs during recessions, is to look for an article that talks about how parents help their kids get jobs at their firms or at other firms in their industry.
- [ ] Search for article talking about parent behavior to help their kids get jobs. Can but does not have to include differences over the business cycle.
